html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Links</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <section>
+        <div>
+        <label for="email">Email</label>
+        <p><input id="email" type="email" name="email"></p>
+      </div>
+      </section>
+
+      <section>
+        <p>
+        <label for="select"></label>
+          <span>Why is coding cool?</span>
+      </p>
+      <select name="cool" id="coolReason">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
```

Comment:
Not a huge deal right now... but the values should be single words if possible, it gets tricky to send spaces and special characters in these `value` attributes.